### PR TITLE
Added Indonesian translation.

### PR DESCRIPTION
**Pull request by artidata**
#### Description of changes

Added translation for the authenticator and cognito error messages.

#### Description of how you validated changes

I'm using it for my website, cross checking it with translation tools.

#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are updated
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/ui/src/i18n/translations.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ import {
   ptDict,
   zhDict,
   svDict,
+  idDict,
```
Comment (wlee221) [inline packages/ui/src/i18n/translations.ts:28]: You should add `idDict` to  `translations` variable in line 76:
https://github.com/aws-amplify/amplify-ui/blob/a9ca442f4d005a597c84a326baf18b4e9bbe78c2/packages/ui/src/i18n/translations.ts#L76-L78

**Review by wlee221 — CHANGES_REQUESTED** [review]
Thanks for your contribution @artidata! Can you: 
- address https://github.com/aws-amplify/amplify-ui/pull/1439/files#r817523250
- Update our docs (this [file](https://github.com/eli6/amplify-ui/blob/main/docs/src/pages/components/authenticator/index.page.mdx#internationalization-i18n), line 417) to include Indonesian as supported language. Languages below English are listed alphabetically.

**Comment by artidata** [comment]
@wlee221, I made the changes. Let me know if I got it right.

**Comment by reesscot** [comment]
@artidata Looking good, just missing docs changes mentioned by @wlee221:

> Update our docs (this [file](https://github.com/eli6/amplify-ui/blob/main/docs/src/pages/components/authenticator/index.page.mdx#internationalization-i18n), line 417) to include Indonesian as supported language. Languages below English are listed alphabetically.

**Comment by artidata** [comment]
@reesscot It's in a separate pull request, because the its merging to different fork by eli6, as that link refers to: https://github.com/eli6/amplify-ui/pull/1/commits. Should it be this file then: https://github.com/aws-amplify/amplify-ui/blob/main/docs/src/pages/components/authenticator/customization.i18n.web.mdx.

**Review by wlee221 — APPROVED** [review]
Looks good, really appreciate your language contribution 🙏

**Review by reesscot — APPROVED** [review]

**Pull request merged**